ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -71,10 +114,27 @@ void WaylandWindow::ApplyPendingBounds() {
   connection_->ScheduleFlush();
 }
 
-void WaylandWindow::Show() {}
+void WaylandWindow::Show() {
+  if (xdg_surface_)
```

Comment:
In Wayland, there is no way to map and unmap windows for showing and hiding them.

One has to destroy xdg_popup and create it again for showing/hiding events